catalog/video/video-admin-plugin/src/test/java/org/codice/alliance/video/ui/TestStreamMonitorHelper.java

```diff
@@ -0,0 +1,145 @@
+
+        }
+
+        @Override
+        public void startMonitoring() {
+
+        }
+
+        @Override
+        public boolean isMonitoring() {
+            return true;
+        }
+    }
+
+    URI uri;
+
+    @Before
+    public void setUp() throws Exception {
+        bundleContext = mock(BundleContext.class);
+        uri = new URI(TEST_URL);
+        List<ServiceReference<StreamMonitor>> serviceReferences = new ArrayList<>();
+        UdpStreamMonitor udpStreamMonitor = mock(UdpStreamMonitor.class);
+        ServiceReference<StreamMonitor> streamMonitorServiceReference =
+                mock(ServiceReference.class);
+        serviceReferences.add(streamMonitorServiceReference);
+
+        when(bundleContext.getServiceReferences(eq(StreamMonitor.class), anyString())).thenReturn(
+                serviceReferences);
+        when(udpStreamMonitor.getTitle()).thenReturn(Optional.of("test"));
+        when(udpStreamMonitor.getStreamUri()).thenReturn(Optional.of(uri));
+        when(bundleContext.getService(any(ServiceReference.class))).thenReturn(udpStreamMonitor);
+        when(streamMonitorServiceReference.getProperty(anyString())).thenReturn(StreamMonitorHelper.SERVICE_PID);
+
+        stream = new StreamMonitorHelper();
+        stream.setContext(bundleContext);
+    }
+
+    @Test
+    public void testStreamMonitorsNoServiceReferences() throws Exception {
+        List<ServiceReference<StreamMonitor>> serviceReferences = new ArrayList<>();
+        when(bundleContext.getServiceReferences(eq(StreamMonitor.class), anyString())).thenReturn(
+                serviceReferences);
+
+        List<Map<String, Object>> list = stream.udpStreamMonitors();
+        assertThat(list, nullValue());
+    }
+
+    @Test
+    public void testStreamMonitorsWrongService() {
+        when(bundleContext.getService(any(ServiceReference.class))).thenReturn(new OtherStreamMonitor());
+        List<Map<String, Object>> list = stream.udpStreamMonitors();
+        assertThat(list, notNullValue());
+        assertThat(list, hasSize(0));
+    }
+
+    @Test
+    public void testStreamMonitors() {
+        List<Map<String, Object>> list = stream.udpStreamMonitors();
+        assertThat(list, notNullValue());
+        assertThat(list.size(), is(1));
+        Map<String, Object> objectMap = list.get(0);
+        assertThat(objectMap.get("id"), is(StreamMonitorHelper.SERVICE_PID));
+        assertThat(objectMap.get(UdpStreamMonitor.METATYPE_TITLE), is("test"));
+        assertThat(objectMap.get(UdpStreamMonitor.METATYPE_MONITORED_ADDRESS), is(TEST_URL));
+    }
+
+    @Test
+    public void testStreamMonitorsNullBundleContext() {
+        StreamMonitorHelper stream = new StreamMonitorHelper();
+        List<Map<String, Object>> list = stream.udpStreamMonitors();
+        assertThat(list, nullValue());
+    }
+
+    @Test
+    public void testStartStreamMonitorWithNoContext() {
+        stream.callStartMonitoringStreamByServicePid(StreamMonitorHelper.SERVICE_PID);
```

Comment:
Added some assertions to these tests.  They are verifying that startMonitoring and StopMonitoring are called from the `StreamMonitorHelper`